### PR TITLE
add press release for registry

### PR DESCRIPTION
**Pull request by susanev**
the link will not work until monday at 5am
@susanev will own merging this pr

### CODE REVIEW

File: themes/default/data/newsroom.toml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 [[releases]]
+date = "18th October 2021"
+title = "Pulumi Ushers in the Developer-first Infrastructure Era with New Package Registry Powering Software Sharing and Reuse"
+summary = "Pulumi Ushers in the Developer-first Infrastructure Era with New Package Registry Powering Software Sharing and Reuse"
+url = "http://info.pulumi.com/press-release/pulumi-registry"
+
 date = "30th September 2021"
```
Comment (cnunciato) [inline themes/default/data/newsroom.toml:7]:
```suggestion
[[releases]]
date = "30th September 2021"
```

Comment (cnunciato) [reply themes/default/data/newsroom.toml:7]: ☝️ I _think_ you need this, as each entry needs to have this same key (so it ends up in the `releases` list).

Comment (susanev) [reply themes/default/data/newsroom.toml:7]: lol sorry and thank you

**Comment by cnunciato** [comment]
@susanev That last check failure is one I've seen once or twice (ever), and I think it's coincided when a running build job was canceled, or a PR was closed while the build was still running, or something that somehow interrupted the job before it could finish normally. Let's see what this next one does, but my hunch is that it may fail again. 🤞

**Comment by susanev** [comment]
@cnunciato should i force push or something?

**Comment by cnunciato** [comment]
Yep, I thought that might fail. Lemme push a commit onto your branch and see if that helps -- I think it should.

**Comment by susanev** [comment]
cool, @cnunciato also mind giving me a green check so i can merge it on monday?

File: themes/default/data/newsroom.toml [hunk 2]
```diff
@@ -1,3 +1,9 @@
+[[releases]]
+date = "18th October 2021"
+title = "Pulumi Ushers in the Developer-first Infrastructure Era with New Package Registry Powering Software Sharing and Reuse"
+summary = "Pulumi Ushers in the Developer-first Infrastructure Era with New Package Registry Powering Software Sharing and Reuse"
```
Comment (susanev) [inline themes/default/data/newsroom.toml:4]:
```suggestion
summary = "Growing collection of Pulumi Packages provides cloud building blocks and reusable cloud architectures for AWS, Azure, Google Cloud, Kubernetes, and top cloud software platforms in 
industry-standard languages"
```

**Pull request merged**